Apps/dashdot/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     ports:
       - "3001:3001" # Port 3001 on the host is mapped to port 3001 in the container
     volumes:
-      - /:/mnt/host:ro # Mounts the root directory of the host as read-only at /mnt/host in the container
+      - /DATA/AppData/$AppID:/mnt/host:ro # Mounts the root directory of the host as read-only at /mnt/host in the container
```

Comment:
If it's not going to mount the root directory, We should probably change the comment. I want people to be able to learn by looking at the comments.

Reply:
You are absolutely right. I apologize for forgetting to update them.

Reply:
No problem, thanks for your assistance with everything!